src/test/shell/bazel/bazel_symlink_test.sh

```diff
@@ -250,7 +314,42 @@ EOF
   [[ "$?" == 1 ]] || fail "unexpected exit code"
```

Comment:
I'm not a fan of these tests that just check that the build failed without seeing why (by writing to $TEST_log and using `expect_log` / `expect_not_log` assertions). These tests could be failing due to a syntax error in the test or other unintended failure. That said, this is a pre-existing issue, so you can leave it alone so long as you've confirmed that this test case produces the expected log output showing that the bazel command fails for the right reasons.

Reply:
Thanks for fixing the pre-existing occurrences too.